### DISCUSSION TITLE
I do not see the option to configure the email provider sendgrid.

### DISCUSSION THREAD

**Discussion by anselmomendes** [strapi/strapi#6486] [Legacy Topics]
[https://ibb.co/w0DkF6b](url)
does not have the icon to configure email.
I installed the provider **"yarn add strapi-provider-email-sendgrid @ beta --save".** help.

**Comment by kshksdrt** [comment]
I do not see the cog icon too. Both before and after adding sendgrid provider.

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question that is not really a bug report or a feature request, questions should be directed to the [discussions](https://github.com/strapi/strapi/discussions). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).

I have converted this to a discussion for you, we try to keep Github clean and only for bug reports and feature requests.
